conf/application.conf

```diff
@@ -198,6 +203,7 @@ microservice {
 # ~~~~~
 cope {
     dwp.originatorId = "DA_PFDWP"
-    feature.enabled = true
-    returnToServiceDays = 91
+    returnToServiceWeeks = 4
 }
+
+ninoHashingKey = "gn:a$V96?cFHmjjcSz31tU,Xe;hOLIYeknIPIbj6Y2GAyht3#Cs+ir&hd{(50`vGAd/k8OIIV:nbM4,c&z9rX+mmK<?qK`sbmKB*Y#Ip.N'$@sX~W[6{KIXA|k`"
```

Comment:
Raise a PR for production so this is different in there

Reply:
https://github.com/hmrc/app-config-production/pull/10990 Done